test/ecto_state_machine_test.exs

```diff
@@ -1,8 +1,9 @@
 defmodule EctoStateMachineTest do
-  use ExUnit.Case, async: true
-  use ExSpec,      async: true
+  use EctoStateMachine.EctoCase, async: true
+  use ExSpec, async: true
 
-  import Dummy.Factories
+  alias EctoStateMachine.User
```

Comment:
I haven't known anything about aliases when start to work on `ecto_state_machine`. So, good catch :+1: